### PR TITLE
(SLV-674) Fix puppet lint errors

### PR DESCRIPTION
**Pull request by johnduarte**
This PR adds back puppet-lint checks that were previously skipped and fixes the errors that those checks revealed.

### CODE REVIEW

**Comment by johnduarte** [comment]
This PR involves moving the loadbalancer puppet manifests.  I have not tested this against a pe_xl deploy to ensure that the loadbalancer classification still works.  I will comment and update the label(s) accordingly after this work is done.

**Comment by johnduarte** [comment]
This PR has been tested with a pe_xl deployment to ensure that moving the load balancer manifests did not break there functionality.  The load balancer was confirmed to be running.

Here is the `nodes.yaml` file showing the hostnames and their roles:
```
 cat build/pe_xl/nodes.yaml
---
groups:
- name: pe_xl_nodes
  config:
    transport: ssh
    ssh:
      host-key-check: false
      user: root
  nodes:
  - name: ip-10-227-3-169.amz-dev.puppet.net
    alias: loadbalancer
  - name: ip-10-227-2-36.amz-dev.puppet.net
    alias: metrics
  - name: ip-10-227-1-156.amz-dev.puppet.net
    alias: compiler_a
  - name: ip-10-227-2-52.amz-dev.puppet.net
    alias: compiler_b
  - name: ip-10-227-2-83.amz-dev.puppet.net
    alias: master
```

Here is a screen shot showing the git sha in use on the master for code manager matches the one in this PR:
slv-674-git-sha

Here is a screen shot showing the load balancer in action:
slv-674-loadbalancer

File: site-modules/userwrapper/manifests/user.pp [hunk 1]
```diff
@@ -13,30 +14,30 @@
     ensure     => present,
     home       => $homedir,
     managehome => false,
-  } ->
+  }
 
-  file { "/var/users/${name}":
+  -> file { "/var/users/${name}":
```
Comment (RandellP) [inline site-modules/userwrapper/manifests/user.pp:19]: Do you know what this arrow does?

**Review by RandellP — APPROVED** [review]

File: site-modules/userwrapper/manifests/user.pp (hunk 1 above)
Comment (johnduarte) [reply site-modules/userwrapper/manifests/user.pp:19]: @RandellP this is one of the chaining operators that puppet has.  They are used to declare that resources should be realized in a defined order.
[Ref](https://puppet.com/docs/puppet/latest/lang_relationships.html#reference-2122)

**Review by billclaytor — APPROVED** [review]

**Pull request merged**